core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairConfiguration.java

```diff
@@ -85,6 +88,11 @@ public long getRepairIntervalInMs()
         return myRepairIntervalInMs;
     }
 
+    public long getNewTableSlideOff()
```

Comment:
Again, this does not relfect the value being returned. There is no "newTableSlideOff" value.